src/Picasso_FieldTypes.hpp

```diff
@@ -77,6 +77,24 @@ struct FieldViewTuple
 
     Views _views;
 
+    // Access by layout.
+    template <class Layout>
+    KOKKOS_INLINE_FUNCTION const auto& get( Layout ) const
```

Comment:
Useful in some applications. I have code where I need to break apart the `Layout` type to satisfy the current `get` functions. This would let me just pass in the layout.